### DISCUSSION TITLE
Is there MuseFX plugin documentation anywhere?

### DISCUSSION THREAD

**Discussion by scorster** [musescore/MuseScore#20807] [General]
https://musescore.org/en/node/358474

**Comment by scorster** [comment]
The silence here (and on .org's forum) is leading me to conclude that no documentation is currently available for various individual MuseFX plugins, like Delay, ProEQ, Compress ...

**Comment by scorster** [comment]
Zendesk inquiry

https://musehub.zendesk.com/hc/en-gb/community/posts/16629853036445-Where-is-the-MuseFX-documentation

**Comment by cbjeukendrup** [comment]
Actually I asked the Muse FX developers about it. They said, the plugins are so simple that they shouldn't need documentation. To which I replied: doubtlessly they are simple if you're familiar with the concept of (for instance) a delay plugin, but if you're not then
- it's interesting to know what the plugin actually does
- those small note buttons at the bottom are pretty mysterious

Unfortunately I didn't receive a reply to that remark. I guess you'd need to look up on the internet what a delay plugin does, and then, based on the documentation of other plugins you'll find, make guesses about how the Muse FX one works.

**Comment by scorster** [comment]
@cbjeukendrup 

Thanks so much for your reply! 

In my opinion it's fairly arrogant that MuseHub has stated, "... the plugins are so simple that they shouldn't need documentation." 

I'm sorry that you, and now we, have had to endure that misguided opinion. If the MuseFX plugins are so simple then creating complete documentation should be simple as well. For the record, Copy/Paste is utterly simple and yet it is frequently documented. It appears that those at MuseHub simply can't be bothered to write some basic documentation. 

And what is self-evident about MuseHub's **Delay plugin?** It goes to 11. (Cute. Yes, many of us saw Spinal Tap.)

It's likely that a majority of MuseScore users know how various delay plugins work, but what about those who don't? Given that the MuseScore notation app, MuseSounds and MuseFX are free, they likely attract beginners who may lack the experience required to understand the various FX plugins and their controls. It's nice when edification accompanies innovation and availability.

The delay plugins I use elsewhere have labeled controls, with time values clearly shown in milliseconds or ticks. Whereas **MuseHub's Delay plugin** has a dumb dial with 21 dots, leading to maximum value of 11. Yes, it's easy to discern that clockwise increases .... um, something. Perhaps the wet/dry level, I guess. And aside from the inscrutable linked icons above the MuseFX logo the plugin's only other control is a menu of cryptically/meaninglessly named presets. 

And once I have a setting I like, can I save it for reuse? No. 

Most other plugins I've used offer the option of saving custom presets—and that would be particularly helpful when flying blind in MuseFX, as it would save me from future guesswork in a familiar darkness. If we can save presets in the current versions that option is not obvious, so it should be documented. 

The **Pro EQ** is obviously a 4-band parametric, with the large knob (at the top of each column of two dials) presumably for gain over a frequency range, but there's no indication of the center of its range nor its Q. The small Focus dial possibly sets the peak of the gain (to some *invisible* amount) OR it attenuates the width of the Q. We don't know. Nor should we have to experiment to find out. 

If an EQ plugin truly needed no documentation it would boast a real-time visual representation of the frequencies to illustrate the effect of the various controls. That would be self explanatory. A set of four unlabeled dials does not provide an adequate picture.  At the very least, frequency labels would help make an undocumented tool more understandable. 

Imagine a one page **Blackhawk Helicopter User Guide** that states:

    Enjoy your new helicopter. 
    Remove it from its crate without damaging.
    Learn to operate by flying it around.
    Provide proper and timely maintenance as needed.

**Comment by Thiago-Morais** [comment]
The lack of documentation is infuriating

**Comment by scorster** [comment]
I'd like to know if the  ProEQ FX plugin makes sense to anyone, so [I posted a test score at .org](https://musescore.org/en/node/358474#comment-1241892)

**Reply by scorster** [reply]
So I guess MuseHub's statement stands, which I paraphrase as:

We are unwilling or unable to provide documentation to unveil the value of our FX plugins ... which lies hidden behind their cryptic/opaque UI.
